tests/src/main/java/org/hibernate/beanvalidation/tck/tests/constraints/builtinconstraints/EmailConstraintTest.java

```diff
@@ -49,17 +47,11 @@ public static WebArchive createTestArchive() {
 	@SpecAssertion(section = Sections.BUILTINCONSTRAINTS, id = "v")
 	public void testEmailConstraint() {
 		Validator validator = TestUtil.getValidatorUnderTest();
-		EmailDummyEntity foo = new EmailDummyEntity();
+		EmailDummyEntity foo = new EmailDummyEntity( "invalid" );
 
 		Set<ConstraintViolation<EmailDummyEntity>> constraintViolations = validator.validate( foo );
-		assertNumberOfViolations( constraintViolations, 0 );
-
-		foo.email = "4 8";
-		constraintViolations = validator.validate( foo );
-		assertCorrectConstraintTypes( constraintViolations, Email.class );
-		assertThat( constraintViolations ).containsOnlyPaths(
-				pathWith()
-						.property( "email" )
+		assertThat( constraintViolations ).containsOnlyViolations(
```

Comment:
I.e. the test should make sure the value is invalid due to it violating the regex() and flags(), but not due to its nature of being an invalid email address (which we cannot assert on).